internal/cronitor/cronitor.go

```diff
@@ -0,0 +1,72 @@
+package cronitor
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	neturl "net/url"
+)
+
+var ErrUnexpectedStatusCode = errors.New("cronitor: received unexpected HTTP status code")
+
+type Monitor struct {
```

Comment:
We should also document the `Monitor` struct.

```suggestion
// Monitor is a ...
type Monitor struct {
```